### PR TITLE
alignment to applications' deployment names

### PR DESCRIPTION
**Pull request by paolocarta**
I aligned the tkn commands to the deployment names specified in the repos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -422,7 +422,7 @@ A `PipelineRun` is how you can start a pipeline and tie it to the git and image
 $ tkn pipeline start build-and-deploy \
     -r git-repo=api-repo \
     -r image=api-image \
-    -p deployment-name=vote-api
```
Comment (ppitonak) [inline README.md:425]: There are PRs in progress renaming the deployments - https://github.com/openshift-pipelines/vote-api/pull/2 and https://github.com/openshift-pipelines/vote-ui/pull/4

@sthaha @praveen4g0 why haven't they been merged yet?

File: 01_pipeline/03_resources.yaml [hunk 2]
```diff
@@ -7,7 +7,7 @@ spec:
   type: git
   params:
   - name: url
-    value: http://github.com/openshift-pipelines/vote-api.git
+    value: http://github.com/paolocarta/vote-api.git
```
Comment (ppitonak) [inline 01_pipeline/03_resources.yaml:10]: Is this change intentional?

File: 01_pipeline/02_update_deployment_task.yaml [hunk 3]
```diff
@@ -1,24 +0,0 @@
-apiVersion: tekton.dev/v1alpha1
```
Comment (ppitonak) [inline 01_pipeline/02_update_deployment_task.yaml:1]: What was your motivation for removing this file?

**Review by ppitonak — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (paolocarta) [reply README.md:425]: The renaming was the only intended change. The rest is just a mistake. Forgot to switch branch for my personal tests.

I have seen those PRs have been merged. Probably this PR does not make sense anymore.

File: 01_pipeline/03_resources.yaml (hunk 2 above)
Comment (paolocarta) [reply 01_pipeline/03_resources.yaml:10]: Sorry, please ignore it. Got it messed up with my local experiments. I will remove the wrong stuff.
